epsagon/trace.py

```diff
@@ -685,23 +685,26 @@ def verify_custom_label(self, key, value):
         :param value: Value for the label data (string)
         :return: True/False
         """
-        if not isinstance(key, str) or not isinstance(value, str):
-            print('EPSAGON: epsagon.label() only supports '
-                  'label keys and values of type string.')
+        if not isinstance(key, str):
+            print('EPSAGON: label key support only string type')
+            print('Received {key}:{value}'.format(key=key, value=value))
```

Comment:
I would just print the type of the key, in the same line (definitely not the value which can be very big).

Reply:
👍